### DISCUSSION TITLE
FS 22 - Bale collection via CP rod

### DISCUSSION THREAD

**Discussion by LoneWolf593** [Courseplay/Courseplay_FS22#1870] [Q&A]
Is it possible for a bale transporter to stop on the descent via CP and AD if it finds that it cannot load any more bales?
I drove the route created by the baler with a horizontal offset of 3.60 m and it also loads excellently, only does not stop when no more bales can be loaded and therefore does not hand over to AD for departure. The function "collecting / wrapping bales" is only of limited use, as vehicles often sink into ditches (NFM) or get stuck in front of trees.

Translation by Google

**Comment by Tensuko** [comment]
That can have many reasons, AD needs to be configured correctly as well, can't tell you whats happening.

**Reply by m0box** [reply]
I do not have an issue with the interaction between CP and AD for the job of collecting bales from a field and delivering them to a point of sale. In fact the job works perfectly until you have more than one autoloader working the field and then there is a risk of the two(+) autoloaders colliding as they go for the same bale. Solution is to set the distance between the autoloaders such that only one (ideally) is working the field at a time. You must have the CP button illuminated/selected on the AD HUD in order for the two to switch back and forth.

My AD vehicle setting is to exit field behind the start (I find this ensures no heading for trees and ditches etc.) and to set the unload fill level low enough to trigger the unload event as CP will hand back to AD when it finds no more bales.
